fearless/Modules/Staking/RecommendedValidators/RecommendedValidatorsViewController.xib

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="17701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
```

Comment:
Are we changing also some styles in this PR?

Reply:
Missed how it slipped into the PR. Rolled back